src/IPub/FormPhone/Controls/Phone.php

```diff
@@ -344,7 +344,7 @@ public function getControl()
 	 *
 	 * @throws Exceptions\InvalidArgumentException
 	 */
-	public function getControlPart()
+	public function getControlPart(): ?Utils\Html
```

Comment:
btw. in nette/forms 2.4.5 are no typehints yet.... they are in 3.0.0-alpha only https://github.com/nette/forms/blob/v3.0.0-alpha1/src/Forms/Controls/BaseControl.php#L274